CONTRIBUTING.rst

```diff
@@ -28,6 +28,32 @@ request" instead (or waiting to request reviewers).
 .. [*] You'd think this would be obvious, but far too many people ignore
        ``README``\ s in practice
 
+Summary of the review process
+===============================
+
+A detailed description can be found in a separate document about the `review
+process <https://cosmo-epfl.github.io/librascal/dev_guide/review_process.html>`_
+
+For developers
+ * We want a clean and tested master branch that is why we review code and use
```

Comment:
```suggestion
 * We want a clean and tested master branch, which is why we review code and use
```
(The alternative is to start a new sentence at "That". See also https://www.quickanddirtytips.com/education/grammar/which-versus-that-0)